independent-projects/bootstrap/runtime/src/main/java/io/quarkus/bootstrap/runtime/QuarkusRuntime.java

```diff
@@ -0,0 +1,143 @@
+package io.quarkus.bootstrap.runtime;
+
+import java.util.Objects;
+
+/**
+ * Provides a low-level API for registering and accessing runtime values generated by Quarkus or Quarkus
+ * extensions. Typically, such values describe a fully configured application, known only at application startup,
+ * such as the HTTP port.
```

Comment:
Is this meant for exposing config options or just about anything and possibly not static?

Reply:
In other words, it's a general purpose value storage?

Reply:
Yes, we can call it that way :)